contentcuration/contentcuration/frontend/channelEdit/components/QuickEditModal/EditBooleanMapModal.vue

```diff
@@ -137,19 +158,16 @@
         }
       },
       async handleSave() {
-        this.validate();
-        if (this.error) {
-          return;
-        }
-
         await Promise.all(
-          this.nodes.map(node => {
+          this.nodes.map(async node => {
             const fieldValue = {};
+            const currentNode = this.getContentNode(node.id);
+
             Object.entries(this.selectedValues).forEach(([key, value]) => {
-              if (value.includes(node.id)) {
-                fieldValue[key] = true;
-              }
+              const existingValue = currentNode[this.field]?.[key] || false;
```

Comment:
This won't allow to remove values from a boolean map when just one node is selected. if I have Categories A and C selected, and want instead Categories A and B, I wont be able to do it.